ios/packages/test-utils/Sources/ui-test/AssetUITestCase.swift

```diff
@@ -50,7 +50,7 @@ open class AssetUITestCase: XCTestCase {
         - mockName: The name of the mock to open
      */
     open func openFlow(_ mockName: String) {
-        navigateToAssetCollection()
+    
```

Comment:
probably shouldnt be removed, we can just not use `super` in the override location